sgp30/sgp30.h

```diff
@@ -35,6 +35,8 @@
 #include "sensirion_common.h"
 #include "sensirion_i2c.h"
 
+#define STATUS_UNKNOWN_FEATURE_SET (-2)
```

Comment:
Rename to `SGP30_ERR_UNKNOWN_FEATURE_SET` and use something safer like (-10) for now. Values down to -3 are already used by SHTs.. I want to come up with a concept on multi-sensor error, but we can still do that later since the API is on the `#define` keyword and not on its value.

Reply:
Done. Let's discuss this further on Friday or some time next week